modules/gathering-data-specific-features.adoc

```diff
@@ -78,6 +78,9 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift4/ose-csi-driver-shared-resource-mustgather-rhel8:v<installed_version_secret_store>`
 |Data collection for the {secrets-store-operator}.
 
+|`registry.redhat.io/lvms4/lvms-must-gather-rhel8:v<installed-version-LVMS>`
```

Comment:
This is only valid for 4.12 and 4.13, 4.14 and up use rhel9 and would fail here

Reply:
We are adding this in 4.12+. Also, rhel8 is mentioned in the image.
Am I missing something here?